nest-forms-backend/src/forms/forms.errors.enum.ts

```diff
@@ -5,6 +5,8 @@ export enum FormsErrorsEnum {
   FORM_NOT_EDITABLE_ERROR = 'FORM_NOT_EDITABLE_ERROR',
   FORM_IS_OWNED_BY_SOMEONE_ELSE_ERROR = 'FORM_IS_OWNED_BY_SOMEONE_ELSE_ERROR',
   FORM_DATA_INVALID = 'FORM_DATA_INVALID',
+  FORM_DEFINITION_NOT_FOUND = 'FORM_DEFINITION_NOT_FOUND',
+  FORM_DEFINITION_GOT_EMAIL = 'FORM_DEFINITION_GOT_EMAIL',
```

Comment:
Please change this to something like "FORM_DEFINITION_NOT_SUPPORTED_TYPE". 
1. There could be more types in the future than email. If another type is created the person creating it would definitely not change this.
2. It is not important the type was email, rather that it is not supported.

Reply:
Yes, I have changed it, also added more info to the message.